### PR TITLE
Now I actually fixed the error and can do what I wanted. 

### PR DESCRIPTION
**Pull request by jamesrez**
No errors anymore, even when I authenticate. Lines 8937 - 8941

I swear this is it.

### CODE REVIEW

File: dist/msrcrypto.js [hunk 1]
```diff
@@ -8934,9 +8934,11 @@ var workerManager = (function () {
 
             // Send the results to the operation object and it will fire
             //   it's onCompleted event.
-            if (op && e.data.type !== "process") {
+            if (op) {
```
Comment (kevlened) [inline dist/msrcrypto.js]: Excellent. This passes all the tests. If you copy it to [`dist/msrcrypto.mjs`](https://github.com/kevlened/msrCrypto/blob/master/dist/msrcrypto.mjs#L8937-L8940), I'll 🚢

Comment (JamieRez) [reply dist/msrcrypto.js]: Done!

**Comment by JamieRez** [comment]
Hey, I finished the minfied js last week. This reply is just to remind you to ship.

File: dist/msrcrypto.js [hunk 2]
```diff
@@ -8933,10 +8933,12 @@ var workerManager = (function () {
             }
 
             // Send the results to the operation object and it will fire
-            //   it's onCompleted event.
-            if (op && e.data.type !== "process") {
+            // it's onCompleted event.
+            if (op) {
                 jobCompleted(worker);
-                op.dispatchEvent(e);
+                if(e.data && e.data.type !== "process"){
```
Comment (kevlened) [inline dist/msrcrypto.js]: Hey @JamieRez, I haven't forgotten about this PR. I just want to ensure this is the logic we want. The alternative is:

```javascript
if(!e.data || e.data.type !== "process")
```

It seems in other places, [the pattern above is used](https://github.com/kevlened/msrCrypto/blob/27815729ceafa1f4e6029fdf0999bd8da83c8752/dist/msrcrypto.js#L8598). I want a better understanding of what this logic is trying to do before merging.

Comment (kevlened) [reply dist/msrcrypto.js]: Your original logic is correct.

The result of the statement is `op.dispatchEvent` is called, [`function opDispatchEvent` calls `processResults(e.data)`](https://github.com/kevlened/msrCrypto/blob/27815729ceafa1f4e6029fdf0999bd8da83c8752/dist/msrcrypto.js#L8634-L8648) and [`processResults` requires `e.data` to be a non-null Object](https://github.com/kevlened/msrCrypto/blob/27815729ceafa1f4e6029fdf0999bd8da83c8752/dist/msrcrypto.js#L8667-L8671).

**Pull request merged**